cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/elements16/IidmToCgmes16.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion.update.elements16;
+
+import com.powsybl.cgmes.conversion.update.IidmChange;
+import com.powsybl.cgmes.conversion.update.IidmToCgmes;
+import com.powsybl.iidm.network.Generator;
+import com.powsybl.iidm.network.Identifiable;
+import com.powsybl.iidm.network.Line;
+import com.powsybl.iidm.network.Load;
+import com.powsybl.iidm.network.ShuntCompensator;
+import com.powsybl.iidm.network.TwoWindingsTransformer;
+import com.powsybl.iidm.network.VoltageLevel;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ */
+public class IidmToCgmes16 {
+
+    public IidmToCgmes findConversion(IidmChange change) {
+        Identifiable o = change.getIdentifiable();
+        if (o instanceof Generator) {
+            return generator;
+        } else if (o instanceof Load) {
+            return load;
+        } else if (o instanceof Line) {
+            return line;
+        } else if (o instanceof TwoWindingsTransformer) {
+            return t2;
+        } else if (o instanceof ShuntCompensator) {
+            return shunt;
+        } else if (o instanceof VoltageLevel) {
+            return vl;
+        } else {
+            return null;
```

Comment:
Add a debug info to display the type of `o` that is not (yet) managed.

Reply:
Done